src/main/scala/org/ergoplatform/nodeView/ErgoNodeViewHolder.scala

```diff
@@ -44,7 +47,37 @@ abstract class ErgoNodeViewHolder[State <: ErgoState[State]](settings: ErgoSetti
   override def postStop(): Unit = {
     log.warn("Stopping ErgoNodeViewHolder")
     history().closeStorage()
-    minimalState().closeStorage
+    minimalState().closeStorage()
+  }
+
+  /**
+    * Tries to apply state to nodeView if possible.
```

Comment:
It do more: publish `SemanticallySuccessfulModifier`/`SemanticallyFailedModification`, create state snapshot

Reply:
@catena2w, done.